charts/coturn/values.yaml

```diff
@@ -24,3 +24,15 @@ securityContext:
 
 coturnTurnListenPort: 3478
 coturnMetricsListenPort: 9641
+
+# This chart optionally supports waiting for traffic to drain from coturn
+# before pods are terminated. Warning: coturn does not have any way to steer
+# incoming client traffic away from itself on its own, so this functionality
+# relies on external traffic management (e.g. service discovery for active coturn
+# instances) to prevent clients from sending new requests to pods which are in a
+# terminating state.
+coturnGracefulTermination: false
+# Grace period for terminating coturn pods, after which they will be forcibly
+# terminated. This setting is only effective when coturnGracefulTermination is
+# set to true.
+coturnGracePeriodSeconds: 86400 # one day
```

Comment:
Since this is a statefulset, by default it would upgrade pods one by one. So, this number will get multiplied by number of instances. Maybe it is better to either make this a `deployment` or set some [`updateStrategy`](https://kubernetes.io/docs/tutorials/stateful-application/basic-stateful-set/#updating-statefulsets) in the statefulset to make things go faster.

Reply:
We can't have more than one instance running at the same time, as they'd bind on the same ports in the host namespace. 

We can probably set `.spec.updateStrategy.rollingUpdate.maxUnavailable` to more than the default (1) to allow faster updates - but on the other hand, the other TURN nodes need to be able to cope with the additional traffic.